client/web/src/enterprise/insights/components/insights-view-grid/components/insight-context-menu/InsightContextMenu.tsx

```diff
@@ -61,14 +62,19 @@ export const InsightContextMenu: React.FunctionComponent<InsightCardMenuProps> =
                             </MenuLink>
 
                             {showYAxisToggleMenu && (
-                                <MenuLink
+                                <MenuItem
+                                    role="menuitemcheckbox"
                                     data-testid="InsightContextMenuEditLink"
-                                    className={classnames('btn btn-outline border-bottom', styles.item)}
-                                    onClick={onToggleZeroYAxisMin}
+                                    className={classnames(
+                                        'd-flex align-items-center justify-content-between',
+                                        styles.item
+                                    )}
+                                    onSelect={onToggleZeroYAxisMin}
+                                    aria-checked={zeroYAxisMin}
                                 >
-                                    <CheckIcon size={16} className={classnames('mr-2', { 'd-none': !zeroYAxisMin })} />{' '}
-                                    Start Y Axis at 0
-                                </MenuLink>
+                                    <input type="checkbox" aria-hidden="true" checked={zeroYAxisMin} onChange={noop} />
```

Comment:
I realized we also need to prevent tab-focusing this as it's just a visual element.
```suggestion
                                    <input type="checkbox" aria-hidden="true" checked={zeroYAxisMin} onChange={noop} tabIndex={-1} />
```